### PR TITLE
Dev smijin doc improvements

### PR DESCRIPTION
**Pull request by SMijin**
Work on issues #18 , #19 , #20 , #21 

Issues #20 and #21 resolved as of opening.

### CODE REVIEW

File: RMK_support/grid.py [hunk 1]
```diff
@@ -20,12 +20,14 @@ def __init__(
         """Grid constructor
 
         Args:
-            xGrid (numpy.ndarray): x coordinates of each spatial grid cell or their widths
-            vGrid (numpy.ndarray): v coordinates of each velocity grid cell or their widths. Defaults to a single cell (effectively no v-grid)
-            lMax (int): Maximum l harmonic number. Defaults to 0.
+            xGrid (numpy.ndarray): x coordinates of each spatial grid cell centre or their widths. If using widths set interpretXGridAsWidths to True.
+            vGrid (numpy.ndarray, optional): v coordinates of each velocity grid cell centres or their widths.  If using widths set interpretXGridAsWidths to True. Defaults to a single cell (effectively no v-grid)
```
Comment (alfieadhemar) [inline RMK_support/grid.py:24]: Should this be interpretVGridAsWidths?

File: RMK_support/grid.py [hunk 2]
```diff
@@ -215,19 +232,26 @@ def __repr__(self) -> str:
     def velocityMoment(
         self, distFun: np.ndarray, momentOrder: int, momentHarmonic=1
     ) -> np.ndarray:
-        """Return velocity moment of distribution function or single harmonic variable
+        """Return velocity moment of distribution function (x,h,v), single harmonic variable (x,v), or velocity space vector (v)
 
         Args:
-            distFun (np.ndarray): Distribution or single harmonic variable values
+            distFun (np.ndarray): Distribution or single harmonic variable values. The
```
Comment (alfieadhemar) [inline RMK_support/grid.py:238]: Remove "The" or finish sentence?

**Review by alfieadhemar — CHANGES_REQUESTED** [review]
Documentation improvements are descriptive and more explicit where needed. Address typos mentioned in review comments before merging

File: RMK_support/grid.py (hunk 1 above)
Comment (SMijin) [reply RMK_support/grid.py:24]: Fixed

File: RMK_support/grid.py (hunk 2 above)
Comment (SMijin) [reply RMK_support/grid.py:238]: Removed

**Comment by SMijin** [comment]
Resolved docs for common_models. Issue #19

**Comment by SMijin** [comment]
Newest commit addresses #18. Ready for review.

**Review by alfieadhemar — APPROVED** [review]

**Pull request merged**